config/www/cgi/script.py

```
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import os
+# import cgi
+# import cgitb
+
+# cgitb.enable()  # Enable debugging
+
+# Define the path to the count file
+count_file = "./tmp/count.txt"
+
+def get_count():
```

Ruff

Missing return type annotation for public function `get_count`

(ANN201)